config/default/jenkins-infra.yaml

```diff
@@ -218,6 +218,23 @@ jenkins:
                         }
                       }
                   - script: >
+                      multibranchPipelineJob('experiment-aws') {
+                        displayName "AWS Experiments"
+                        branchSources {
+                          github {
+                            id('2020120401')
+                            scanCredentialsId('github-access-token')
+                            repoOwner('jenkins-infra')
+                            repository('experiment-aws')
+                          }
+                        }
+                        factory {
+                          workflowBranchProjectFactory {
+                            scriptPath('Jenkinsfile_k8s')
```

Comment:
I don't see a k8s file in https://github.com/jenkins-infra/experiment-aws

Reply:
I think it would be better to directly use a real repository named like "jenkins-infra/aws" and mentioned the experimental status in the README so we don't do the work two time